### PR TITLE
New labels

### PR DESCRIPTION
**Pull request by burakukula**
They will replace Badge (currently we use only mustard badge on prod, so this will be quick migration)

Screenshot 2019-12-23 at 15 03 20
Screenshot 2019-12-23 at 15 03 28

labels mov

### CODE REVIEW

File: src/components/labels/Label.jsx [hunk 1]
```diff
@@ -0,0 +1,183 @@
+// @flow strict
+
+import React from 'react';
+import type {Node} from 'react';
+import classNames from 'classnames';
+import Text from '../text/Text';
+import * as IconModule from '../icons/Icon';
```
Comment (matzimowski) [inline src/components/labels/Label.jsx]: import Icon, {TYPE: ICON_TYPE} from '../icons/Icon';`?

File: src/components/labels/Label.jsx [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+  /**
+   * Children to be rendered inside Label
+   * @example <Label
+   *           type="default"
+   *           color="blue"
+   *          >
+   *            example label
+   *          </Label>
+   */
+  children: Node,
```
Comment (matzimowski) [inline src/components/labels/Label.jsx]: should we allow to define children? i know we did in the past, but i know not sure if it's needed here now. wdyt?

File: src/components/labels/Label.jsx (hunk 1 above)
Comment (burakukula) [reply src/components/labels/Label.jsx]: 🧐

Comment (burakukula) [reply src/components/labels/Label.jsx]: @matzimowski This import would not work I think, there is no export of `ICON_TYPE`. I need to also use `IconTypeType` which I get from `IconModule `. I saw we do it this way in `IconAsButton`. But maybe there is a better way...

Comment (matzimowski) [reply src/components/labels/Label.jsx]: ok, let's stick to what we have. 👌

**Pull request merged**